analysis/anomaly_score.py

```diff
@@ -70,12 +74,18 @@ def get_anom_avg(model_name, new_null_models, img_path, method, lpips_net):
 
     anom_avg = 0
     _, png_path1 = visualization_path(model_name, method, img_path)
-    img1 = lpips.im2tensor(lpips.load_image(str(png_path1))).to(device)
+    img1 = load_image_tensor(png_path1).to(device)
     for null_m in new_null_models:
         _, png_path2 = visualization_path(null_m, method, img_path)
-        img2 = lpips.im2tensor(lpips.load_image(str(png_path2))).to(device)
+        img2 = load_image_tensor(png_path2).to(device)
 
         # compute distance and add to anom_avg
         anom_avg += loss_fn.forward(img1, img2).item()
     anom_avg /= num_null
     return anom_avg
+
+
+def load_image_tensor(path: Path):
+    if not path.exists():
+        raise RuntimeError(f"Path doesn't exist: {path}")
```

Comment:
this is just for better error messages (otherwise it fails later in a way that doesn't show you the path)